recipes/sortmerna/2.0/meta.yaml

```diff
@@ -9,11 +9,10 @@ source:
 
 build:
   number: 1
-  skip: True # [not linux64 or osx]
 
 requirements:
   build:
-    - gcc
+    - gcc # [linux]
     - zlib
   run:
     - libgcc
```

Comment:
libgcc needs also to be garded only for linux and for OSX your need to add llvm # [osx] (ideally). Please see your guidelines.